### ISSUE TITLE
Create Journal references page

### ISSUE THREAD

**Issue by naushinthomson** [elifesciences/schematron-gitbook#72]
- [x] Create page 
- [x] Add description of what is being covered
- [x] Add description of checks required on this content, compiled from current protocol documents
- [x] Add examples covering correct and incorrect scenarios, as required to support checks
- [x] Populate with schematron messages
- [x] Provide explanations for how to fix schematron errors and warnings
- [x] Review house style document to ensure all requirements are covered by GitBook and schematron
- [x] Review schematron messages related to this topic to ensure they are user friendly
- [x] Becky approved page
- [x] Melissa approved page
- [x] Fred approved page
- [x] James approved page
- [x] Naushin approved page
- [x] Update Kriya queries as necessary
- [x] Share page with Exeter and invite feedback
- [x] Revise with Exeter's suggestions

### Definition of done
- [x] Page completed
- [x] Team reviewed and approved page?
- [x] Assign Fred to update the Schematron

**Comment by naushinthomson** [comment]
https://app.gitbook.com/@elifesciences/s/productionhowto/article-details/content/references/journal-references
This is now ready to review! I think I will admit defeat with the gitbook commenting system and ask for any comments to be left on this issue. However, if there are any spelling/grammar errors please just feel free to edit and merge! I'm sure I must be missing something because the actual guidance part is quite short...

**Comment by FAtherden-eLife** [comment]
> Authors: Enter each author name as a surname followed by initials.

Change surname to surname(s)

**Comment by FAtherden-eLife** [comment]
> e-location ID: The e-location identifier for the article (if applicable).
> First page/last page: The page number on which the article starts and ends (if applicable). Some page numbers will be prefixed with letters.

Is it worth clarifying here (and in the table) that it's one or the other? I know it's mentioned further down but might be good to add for consistency.

**Comment by FAtherden-eLife** [comment]
> If any of the mandatory information is not provided, please leave the following author query, replacing 'XXXXXX' with the missing information:
> Please provide the XXXXXX for this reference.

Even if it can be found with a google search?

**Comment by FAtherden-eLife** [comment]
> The default fields in Kriya will need to be altered to include the following information in this order:

There's no mention of 'in press' in this section or above it.

**Comment by FAtherden-eLife** [comment]
> Once you have entered the information in the fields, click on 'Search CrossRef'. This will compare the information entered to the CrossRef database and give you a variety of articles to choose from.

I always search pubmed first (especially if there's a pmid). Is that the right thing to do? If so, do we need to update this section?

**Comment by FAtherden-eLife** [comment]
> Many journals do not use volume numbers, page ranges or e-locations.

Suggest changing 'Many' to 'Some'. This isn't the norm and we should be aiming to include this information as much as possible (where it exists).

**Comment by FAtherden-eLife** [comment]
> Research-gate-check

As discussed, this will also for for researchgate (no space).

**Comment by FAtherden-eLife** [comment]
> article-title-fullstop-check-1

This doesn't happen often anymore (perhaps because of this check), but it was also intended to catch partial journal names included in the article title, e.g. 'Reversal of prolonged obesity-associated cerebrovascular dysfunction by inhibiting microglial Tak1. _Nature'_. _Neuroscience_. Might be worth including in the action to check that the journal name (or part of it) is not included in the article title.

**Comment by FAtherden-eLife** [comment]
> If the authors refer to both the corrected/retracted article and the original article, or if they mention the citation is to a corrected/retracted article in the context, you can leave this reference in.

Should we include something about whether there is another reference to the article itself? (being explicit about how the authors have 'refer'ed to it)

**Comment by FAtherden-eLife** [comment]
> If the original submitted manuscript cites the corrected/retracted paper, you can leave this reference in, but leave the following author query:

Is it worth specifying that this should be done at pre-author stage(s) or is that implicit?

**Comment by FAtherden-eLife** [comment]
> Action: This warning will appear if a journal reference has an article title with formatting applied to the whole content (e.g. if the title is entirely in italics or bold formatting). Look up the reference online to see how it should be formatted and update accordingly. If you can't find the reference online, leave the following query:
>> Please provide the full bibliographic details for this reference.

Is this the correct author query?

**Comment by FAtherden-eLife** [comment]
For eloc-page-assert and volume-assert is it worth clarifying that this is often because an article has been published online before a print version (and so doesn't have those details yet)?

**Comment by FAtherden-eLife** [comment]
> journal-preprint-check
> ... biorxiv, arxiv, chemrxiv, peerj preprints, psyarxiv, paleorxiv, preprints ...

Any suggestions (from anyone) about what others we should include for this

**Comment by FAtherden-eLife** [comment]
> elife-ref-check
> Action: This error will appear if a journal reference has the journal name 'eLife' but the doi is incorrectly formatted. eLife articles have dois in the format: 10.7554/eLife.XXXXX. Look up the article online to check what the doi should be.

What should Exeter do if this has not been published yet?

**Comment by FAtherden-eLife** [comment]
> Look up the article online to check whether it has been published in a journal, or whether it is a paper from a conference. If the conference publishes its papers in a journal, keep this as a journal reference, otherwise change it to a conference reference (hyperlink to be added).

Is it worth clarifying how to identify that a 'conference publishes its papers in a journal'? For example if it is indexed by pubmed, and how to find that out.

**Comment by FAtherden-eLife** [comment]
> online-journal-w-page
> PLOS, eLife or mBio

Any suggestions (from anyone) about what other online-only journals we should include in this test?

**Comment by FAtherden-eLife** [comment]
Can we include a collab and In press example in the XML structure please?

**Collab**
```xml
<element-citation publication-type="journal">
  <person-group person-group-type="author">
    <name>
      <surname>Blake</surname> <given-names>JA</given-names>
    </name>
    <name>
      <surname>Bult</surname> <given-names>CJ</given-names>
    </name>
    <name>
      <surname>Eppig</surname> <given-names>JT</given-names>
    </name>
    <name>
      <surname>Kadin</surname> <given-names>JA</given-names>
    </name>
    <name>
      <surname>Richardson</surname> <given-names>JE</given-names>
    </name>
    <collab>the Mouse Genome Database Group</collab>
  </person-group>
  <year iso-8601-date="2009">2009</year>
  <article-title>The mouse genome database genotypes::phenotypes</article-title>
  <source>Nucleic Acids Research</source>
  <volume>37</volume>
  <fpage>D712</fpage>
  <lpage>D719</lpage>
  <pub-id pub-id-type="doi">10.1093/nar/gkn886</pub-id>
  <pub-id pub-id-type="pmid">18981050</pub-id>
</element-citation>
```

**In press**
```xml
<element-citation publication-type="journal">
  <person-group person-group-type="author">
    <name>
      <surname>Wu</surname> <given-names>C</given-names>
    </name>
    <name>
      <surname>Liu</surname> <given-names>Y</given-names>
    </name>
    <name>
      <surname>Yang</surname> <given-names>Y</given-names>
    </name>
    <name>
      <surname>Zhang</surname> <given-names>P</given-names>
    </name>
    <name>
      <surname>Zhong</surname> <given-names>W</given-names>
    </name>
    <name>
      <surname>Wang</surname> <given-names>Y</given-names>
    </name>
    <name>
      <surname>Wang</surname> <given-names>Q</given-names>
    </name>
    <name>
      <surname>Xu</surname> <given-names>Y</given-names>
    </name>
    <name>
      <surname>Li</surname> <given-names>M</given-names>
    </name>
    <name>
      <surname>Li</surname> <given-names>X</given-names>
    </name>
    <name>
      <surname>Zheng</surname> <given-names>M</given-names>
    </name>
    <name>
      <surname>Chen</surname> <given-names>L</given-names>
    </name>
    <name>
      <surname>Li</surname> <given-names>H</given-names>
    </name>
  </person-group>
  <year iso-8601-date="2020">2020b</year>
  <article-title>Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</article-title>
  <source>Acta Pharmaceutica Sinica B</source>
  <comment>In press</comment>
  <pub-id pub-id-type="doi">10.1016/j.apsb.2020.02.008</pub-id>
</element-citation>
```

**Comment by FAtherden-eLife** [comment]
How do we want to handle tests which apply to all types of reference? In a general citation page?

**Comment by FAtherden-eLife** [comment]
Sorry for all the comments - that's me done. Great job on this @naushinthomson.

**Comment by Melissa37** [comment]
>The following information can be added for journal references:
Table

>Kriya field

Please add '(s)" to Author and Collaboration entries, eg Author(s) Collaboration(s)

>Mandatory? 

Author and Collaboration entries please change to:

>Yes (and/or a collaboration must be entered instead)
>Yes (and/or an author(s) must be entered instead)

To indicate can have both.

**Comment by Melissa37** [comment]
>The following information can be added for journal references:
Table

>Mandatory?

for e-location ID and First page entries please indicate you cannot have both but it is preferred to have one of them - few exceptions
Please also indicate if you have a First page it is usual to have a Last page but not mandatory

**Comment by Melissa37** [comment]
>Where the optional information is available, this should also be added in. 

What does this mean? We do not expect manual checking of references so I am not sure there is value in adding this statement.
The non-mandatory fields are not really optional, it's just that we cannot always include them.

Perhaps change to:

>non-mandatory fields are required if they exist. Validation of references via PubMed or Crossref API tools should pull them in if the author did not provide them. Schematron warnings may also pick them up and should be checked at source.

**Comment by Melissa37** [comment]
>Journal name: The name of the journal in which the article was published.

Should we indicate we use full journal title and not an abbreviated form?

**Comment by Melissa37** [comment]
>Volume: The volume of the journal in which the article was published (if applicable).

Perhaps mention that we do not publish Issue numbers. Some publishers do, so for clarity makes sense to indicate we actively don't

**Comment by Melissa37** [comment]
>e-location ID: The e-location identifier for the article (if applicable).

Suggest changing to:

>e-location ID: The e-location identifier for the article (if applicable; most journal references will have either page numbers or an e-location ID).

>First page/last page: The page number on which the article starts and ends (if applicable). Some page numbers will be prefixed with letters.

Suggest changing to:

>First page: The page number on which the article starts (if applicable; most journal references will have either page numbers or an e-location ID). Some page numbers will be prefixed with letters.
>Last page: The page number on which the article ends (if applicable; some articles are one-page long and so an end page is not required as it is the same as the first page). Some page numbers will be prefixed with letters.

**Comment by Melissa37** [comment]
>PMID: The PubMed Identifier for the article (if applicable)

Suggest changing to:

>PMID: The PubMed Identifier for the article (if applicable). This will not be supplied by the author but is added during pre-editing via the PubMed API tool

**Comment by Melissa37** [comment]
>The default fields in Kriya will need to be altered to include the following information in this order:
Author 
Collaboration (if appropriate)
Year 
Article title
Journal name
Volume (if available)
e-location ID or first and last page (if available)
DOI (if available)
PMID (if available)

Suggest changing to:

>The default fields in Kriya will need to be altered to include the following information in this order:
Author/Collaboration (as appropriate, in order as provided in citation)
Year 
Article title
Journal name
Volume (if available)
e-location ID or first and last page (if available)
DOI (if available)
PMID (if available)

Please note, to delete an unnecessary field click the red 'bin' button and to add a field click the green '+' button to the entry above where you need to add it.

**Comment by Melissa37** [comment]
>Often you don't need to fill in all the details for a journal reference - filling in some of the fields and then clicking on the 'Search CrossRef' button will usually give you the correct article. The rest of the details will then be automatically filled in when you click on the article. 

Which fields need to be filled in to get the correct result and auto population of the rest of the fields?

**Comment by Melissa37** [comment]
>Watch out for unexpanded journal names when trying to validate them. Sometimes, the authors may have provided the contracted version of the journal name, e.g. 'Mol Biol Evo' for 'Molecular Biology and Evolution' (they will likely take this from a source like PubMed which shows the contracted version by default). If this is no expanded to the full version of the name, the validator may not be able to pick up the reference correctly to assign DOI/PMID.

Arghhhh!! This needs to be fed back to Exeter!! :-)

**Comment by Melissa37** [comment]
>Look out for journal names that are incorrectly run on to the end of article titles. This is often caught by the schematron rules but in some cases, it may not be.

This indicates we are expecting someone to manually run their eye across all references...is that right?

**Comment by Melissa37** [comment]
>Journals are not standardised in the way they archive their articles.

This example is of ahead of print. When this journal publishes the VoR they have page ranges and volume info (eg https://www.liebertpub.com/doi/10.1089/jwh.2019.7821).

I think this needs to be more explicit. Most examples of this are ahead of print publication aren't they? (Our name for this is PoA)

**Comment by Melissa37** [comment]
>err-elem-cit-journal-2-1
Error: Each <element-citation> of type 'journal' must contain one and only one <person-group> element. Reference 'XXXXXX' has XXXXXX <person-group> elements.

Could this fire if there are more than 1 author groups in the XML?

**Comment by Melissa37** [comment]
>err-elem-cit-journal-2-2
Error: Each <element-citation> of type 'journal' must contain one <person-group> with the attribute person-group-type 'author'. Reference 'XXXXXX' has a <person-group> type of 'XXXXXX'.
Action: This error will appear if there is no author list, or if authors have been tagged as other kinds of contributors (e.g. editors or curators). If there is no author list, make sure this is added by searching for the article details online. If there are contributors other than authors or collaborations, these should be removed.

Should they just be removed? Would it be best to check whether they should be listed as authors first? Also, how can this happen in Kriya if the options are author or collaborator? Would you have to delete them and then re-add them if they should be there?

**Comment by Melissa37** [comment]
>err-elem-cit-journal-3-1
Error: Each <element-citation> of type 'journal' must contain one and only one <article-title> element. Reference 'XXXXXX' has XXXXXX <article-title> elements.
Action: This error will appear if a journal reference has more than one or no article titles. If the authors have not provided the article title but have given other bibliographic details (e.g. journal, volume, page range or elocation ID) it may be possible to search for the article online. If there are not enough details to find the article title, add the following author query:

Please add instructions on what to do if there are 2 article-titles tagged

**Comment by Melissa37** [comment]
>err-elem-cit-journal-4-1
Error: Each <element-citation> of type 'journal' must contain one and only one <source> element. Reference 'XXXXXX' has XXXXXX <source> elements.
Action: This error will appear if a journal reference has more than one or no journal name. Check that the journal name has not been accidentally captured as part of the article title. If the authors have not provided the journal name but have given other bibliographic details (e.g. article title, volume, page range or elocation ID) it may be possible to search for the article online. If there are not enough details to find the article, add the following author query:

Suggest changing to
> Check that the journal name has not been accidentally captured as part of the article title (which will result in no source) or that the journal title or article title has not be split out over multiple source entries...

**Comment by Melissa37** [comment]
>err-elem-cit-journal-5-1-3
Error: There may be no more than one <volume> element within a <element-citation> of type 'journal'. Reference 'XXXXXX' has XXXXXX <volume> elements.
Action: This error will appear if a journal reference has more than one volume number. Remove the extra volume number(s) and make sure that a page number/e-location ID has not been tagged as a volume by mistake.

Suggest changing part of the action from:
>Remove the extra volume number(s) and make sure that a page number/e-location ID has not been tagged as a volume by mistake.

To:

>Check the citation details and remove the incorrect volume number(s). Make sure that a page number/e-location ID has not been tagged as a volume by mistake.

**Comment by Melissa37** [comment]
There is no mention of in press in the explanatory text at the top of the article

**Comment by Melissa37** [comment]
>err-elem-cit-journal-5-1-2
Error: A <volume> element within a <element-citation> of type 'journal' must contain at least one character and may not contain child elements. Reference 'XXXXXX' has too few characters and/or child elements.
Action: This error will appear if the volume field in a journal reference is empty, or if there are formatting elements present (e.g. bold, underline). Make sure the correct volume number is entered if appropriate by looking up the article details online, and make sure the contents do not contain any formatting.

This text should be added to most of the intrusions or removed from this one:

> if appropriate by looking up the article details online,

**Comment by Melissa37** [comment]
>err-elem-cit-journal-6-2
Error: If <fpage> is present, neither <elocation-id> nor <comment>In press</comment> may be present. Reference 'XXXXXX' has <fpage> and one of those elements.
Action: This error will appear if a journal reference has a first page and an e-location ID or 'Inpress' comment. Look the article up online and make sure that the page range is correct. If the article in fact has an e-location ID rather than a page range, remove any first or last pages and enter the ID in the e-location ID field. If the article is published, any 'In press' tag/comment should be removed. If the article has not been published, leave the following author query:
Please check whether this In press reference has now been published and, if so, please provide the full citation details.
If the authors do not have a publication date or if the publication date will be more than a few days after the publication of the eLife article, remove the page range and keep the 'In press' comment.

...or 'Inpress' comment. should be corrected to In press comment

**Comment by Melissa37** [comment]
>err-elem-cit-journal-6-2
Error: If <fpage> is present, neither <elocation-id> nor <comment>In press</comment> may be present. Reference 'XXXXXX' has <fpage> and one of those elements.
Action: This error will appear if a journal reference has a first page and an e-location ID or 'Inpress' comment. Look the article up online and make sure that the page range is correct. If the article in fact has an e-location ID rather than a page range, remove any first or last pages and enter the ID in the e-location ID field. If the article is published, any 'In press' tag/comment should be removed. If the article has not been published, leave the following author query:
Please check whether this In press reference has now been published and, if so, please provide the full citation details.
If the authors do not have a publication date or if the publication date will be more than a few days after the publication of the eLife article, remove the page range and keep the 'In press' comment.

Suggest changing action to:

>Action: This error will appear if a journal reference has a first page and an e-location ID and/or 'In press' comment. It can only have one of these. Please check and update the reference.
If the article has not been published, leave the following author query:
Please check whether this In press reference has now been published and, if so, please provide the full citation details.
If the authors do not have a publication date or if the publication date will be more than a few days after the publication of the eLife article, remove the page range and keep the 'In press' comment.

**Comment by Melissa37** [comment]
>err-elem-cit-journal-6-3


Same comments as above for err-elem-cit-journal-6-2

**Comment by Melissa37** [comment]
>err-elem-cit-journal-6-5-2
Error: <lpage> must be larger than <fpage>, if present. Reference 'XXXXXX' has first page <fpage> = 'XXXXXX' but last page <lpage> = 'XXXXXX'.
Action: This error will appear if a journal reference has a first page value larger than the last page value (e.g. first page: 27, last page: 20). Look up the article online to identify the appropriate page range information and correct the reference. If the article in fact has an e-location ID rather than a page range, remove any first or last pages and enter the ID in the e-location ID field.

This will usually occur if the author truncates the last page, eg 123-34.
I'd reword this Action using that as an example instead.

**Comment by Melissa37** [comment]
>Error: Comment elements with content other than 'In press' are not allowed. Reference 'XXXXXX' has such a <comment> element.
Action: This error will appear if the 'Comment' field in Kriya has anything other than 'In press'. If the comment element contains relevant bibliographic information (article title, journal name, volume, etc) move this information to the correct fields and delete the comment element. If the comment contains a variant on 'in press' (e.g. 'in production', 'forthcoming' etc), correct this to 'In press' and leave the following author query:
 

>If the comment contains a variant on 'in press' 

Change to:

>If the comment contains a variant on 'In press'

**Comment by Melissa37** [comment]
>Please update this reference if this article is now in press. If it is still in preparation, it will need to be removed from the reference list
If the authors respond to say the article is now in press, update the 'Comment' field in Kriya with the text 'In press'. If the authors respond to say the article is still in preparation, the reference will need to be removed from the reference list and cited in the text as, for example 'Smith et al., in preparation'.

I think we need to update the AQ to:

>Please update this reference if this article is now in press. If it is still in preparation, it will need to be removed from the reference list and cited in the text as, for example 'Smith et al., in preparation'.

**Comment by Melissa37** [comment]
>err-elem-cit-journal-10
Error: If <pub-id pub-id-type="pmid"> is present, the content must be all numeric. The content of <pub-id pub-id-type="pmid"> in Reference 'XXXXXX' is XXXXXX.
Action: This error will appear if a PubMed ID has any letters. PMIDs should only contain numbers - look up the article online to get the correct ID and enter this.

Suggest changing the action to:

>Action: This error will appear if a PubMed ID has any letters. PMIDs should only contain numbers - search for the article on PubMed [https://pubmed.ncbi.nlm.nih.gov/] to get the correct ID and enter this.

**Comment by Melissa37** [comment]
>Action: This error will appear if a journal reference has the doi for PNAS but the journal name has been entered as something other than 'PNAS'. Firstly, make sure that following the DOI leads to the same article as the reference in Kriya. When you have confirmed this, apply the correct capitalisation for the journal name. For example, 'Proceedings of the National Academy of Sciences, Proceedings of the National Academy of Sciences of the United States of America' should all be changed to 'PNAS'.

Suggest changing to:

Action: This error will appear if a journal reference has the doi prefix for PNAS (10.1073/pnas) but the journal name has been entered as something other than 'PNAS'. Firstly, make sure that following the DOI leads to the same article as the reference in Kriya. When you have confirmed this, apply the correct capitalisation for the journal name. For example, 'Proceedings of the National Academy of Sciences, Proceedings of the National Academy of Sciences of the United States of America' should all be changed to 'PNAS'.

**Comment by Melissa37** [comment]
>PNAS
Error: ref 'XXXXXX' has the doi for 'PNAS' but the journal name is XXXXXX, which is incorrect.
Action: This error will appear if a journal reference has the doi for PNAS but the journal name has been entered as something other than 'PNAS'. Firstly, make sure that following the DOI leads to the same article as the reference in Kriya. When you have confirmed this, apply the correct capitalisation for the journal name. For example, 'Proceedings of the National Academy of Sciences, Proceedings of the National Academy of Sciences of the United States of America' should all be changed to 'PNAS'.

Why do we ask to check whether "the DOI leads to the same article as the reference in Kriya' for PNAS but don't do it for PLOS or the BMJ?

**Comment by Melissa37** [comment]
>G3
Error: ref 'XXXXXX' has the doi for 'G3' but the journal name is XXXXXX - it should be either 'G3: Genes|Genomes|Genetics' or 'G3: Genes, Genomes, Genetics'.
Action: This error will appear if a journal reference has the journal name 'G3' formatted incorrectly. Due to an issue with how G3 supplies its data to PubMed/Crossref, the journal name will sometimes have the unicode character replacing the ':'. Change the journal name to 'G3: Genes|Genomes|Genetics' or 'G3: Genes, Genomes, Genetics'.

Why do we allow 'G3: Genes|Genomes|Genetics' AND 'G3: Genes, Genomes, Genetics'.
Should it not just be one of these variants?

**Comment by Melissa37** [comment]
>ampersand-check
Warning: ref 'XXXXXX' appears to contain the text 'amp', is this a broken ampersand?
Action: This error will appear if a journal reference contains the text 'amp'. Look up the article online to check whether this is a broken ampersand and fix if so. 

Should the action be:
>This error will appear if a journal reference contains the text 'amp'. This usually indicates a conversion error where '&' has converted to the text string 'amp'. Please check the source and correct as necessary - if the source used & please replace amp with &; if the source used 'and' please update to 'and'

**Comment by Melissa37** [comment]
>Research-gate-check
Error: ref 'XXXXXX' has a source title 'XXXXXX' which must be incorrect.
Action: This error will appear if a journal name has been entered as 'Research Gate' (or 'research gate' or any other capitalisation alternatives). Check what the correct journal name should be by looking up the article online and enter that instead. 

Does this happen if an author has found a PDF to a research article on Research Gate and so citing that instead of the source? If so, I think this action should be clearer as it could be that it's hard to find the real source and the user might just find Research Gate again?

**Comment by Melissa37** [comment]
>journal-replacement-character-presence
Error: element contains the replacement character '�' which is unallowed - XXXXXX
Action: This error will appear if a journal name contains the character ‘�’. This usually indicates that processing has gone wrong, or it has been used to replace an unknown, unrecognized or unrepresentable character. Look up the article online and make sure the fields have all been entered correctly.

Is this just searching in the journal name or anywhere in the reference? @FAtherden-eLife 

Also suggest changing to '...indicates a conversion error has occurred and replaced an unknown, unrecognized or unrepresentable character with  ‘�’.  Refer to the source of the reference and update with the correct character.

**Comment by Melissa37** [comment]
>journal-off-presence
Warning: ref 'XXXXXX' has a source title which contains the text 'official journal' - 'XXXXXX'. Is this necessary?
Action: This warning will appear if a journal name includes the text 'official journal'. Delete this text from the journal name. 

Can we do this categorically?

**Comment by Melissa37** [comment]
General

>Look up the article online

Can we change this to something like find the source for the journal title (either PubMed, PMC or the publisher website)?

**Comment by Melissa37** [comment]
>article-title-fullstop-check-3
Warning: ref 'XXXXXX' has an article-title which ends with some full stops - is this correct? - XXXXXX
Action: This error will appear if a journal reference has an article title which ends with multiple full stops. The full stops should be removed as this will make the journal reference incorrectly formatted. 

Should there never be fullstops at the end of the article-title?

If so, I'd change this to:

> Action: This error will appear if a journal reference has an article title which ends with 1+ full stops. All the full stops should be removed.

**Comment by Melissa37** [comment]
>article-title-correction-check
Warning: ref 'XXXXXX' has an article-title which begins with 'Correction', 'Retraction' or 'Erratum'. Is this a reference to the notice or the original article?
Action: This warning will appear if a journal reference has an article title beginning with 'Correction', 'Retraction' or 'Erratum'. Read the context surrounding the citation of this reference to work out what to do. If the authors refer to both the corrected/retracted article and the original article, or if they mention the citation is to a corrected/retracted article in the context, you can leave this reference in. Otherwise, check the original submitted manuscript to make sure the wrong details have not been entered for this reference. If the original submitted manuscript cites the corrected/retracted paper, you can leave this reference in, but leave the following author query:
This reference is to a correction/retraction/erratum. Please confirm whether this is correct.

I'd like to add some context somewhere about why we do this - it's a Kriya bug right?

**Comment by Melissa37** [comment]
>article-title-journal-check
Warning: ref 'XXXXXX' has an article-title which contains the text ' journal '. Is a journal name (source) erroneously included in the title? - 'XXXXXX'
Action: This warning will appear if a journal reference has an article title which includes the text 'journal'. This may need to be removed if, for example it reads 'Journal: PNAS'. However, if the word 'journal' is part of the journal name, then it needs to be kept (e.g. Journal of Neuroscience). 

I sense the interpretation of this is wrong; it's checking the article title, not the journal title, as this indicates: 'However, if the word 'journal' is part of the journal name, then it needs to be kept (e.g. Journal of Neuroscience).' I think the right interpretation should be to find a journal article title that references another journal?

@FAtherden-eLife

**Comment by Melissa37** [comment]
>article-title-child-1
Warning: ref 'XXXXXX' has an article-title with one child XXXXXX element, and no text. This is almost certainly incorrect. - XXXXXX
Action: This warning will appear if a journal reference has an article title with formatting applied to the whole content (e.g. if the title is entirely in italics or bold formatting). Look up the reference online to see how it should be formatted and update accordingly. If you can't find the reference online, leave the following query:
Please provide the full bibliographic details for this reference.

I don't understand why for this action we introduce the concept you might not be able to find a reference online, but it's not in any of the other actions.

Also, I think you should be able to just remove the formatting for the article title and not have to check it?

**Comment by Melissa37** [comment]
>a-title-replacement-character-presence

See comments on journal-replacement-character-presence

**Comment by Melissa37** [comment]
>online-journal-w-page
Error: XXXXXX is a XXXXXX article, but has a page number, which is incorrect. 
Action: This error will appear if a journal reference has the journal name PLOS, eLife or mBio and a page number. These journals never use page numbers for their articles, so any page number fields should be removed.

These journals always have elocation-IDs so the action is not just to remove the page numbers but to add an eLocation ID too

**Comment by Melissa37** [comment]
>journal-doi-test-1
Warning: XXXXXX is a journal ref without a doi. Should it have one?
Action: This warning will appear if a journal reference does not have a DOI. Revalidate the reference first, to ensure it is not just a system glitch. Then, look up the article online and check PubMed and Crossref as well, to see whether this information can be found. If the article has a PubMed ID but no DOI, the reference can be left as it is. Some articles will not have DOIs. This is particularly common with older articles e.g. those published before ~1990, but some journals also do not assign DOIs to their articles (e.g. Development). While doing this check, keep an eye out for differing article information. If the author names or the title are slightly different from the published details or what is recorded in PubMed, the validator will not identify the reference.

Suggest changing the action to:

>This warning will appear if a journal reference does not have a DOI. Revalidate the reference first, to ensure it is not just a system glitch. If it is still not added look up the article on PubMed , if the article is listed and no DOI is present, the reference can be left as it is. If it is not on PubMed look at the journal source online, or use Crossref to check whether there should be a DOI.
Some articles will not have DOIs. This is particularly common with older articles e.g. those published before ~1990, but some journals still do not assign DOIs to their content now (e.g. _Development_). eLife Schematron attempts to filter out false positives so please inform Production staff of this message on content that does not have DOIs (for example _Development_ references do not fire this warning).
While doing this check, keep an eye out for differing article information. If the author names or the title are slightly different from the published details or what is recorded in PubMed, the validator will not identify the reference.

**Comment by Melissa37** [comment]
Nice job @naushinthomson Thanks!! Another massive page with loads of warnings covered :-)

**Comment by JGilbert-eLife** [comment]
I'm not sure the 'Examples of journal citations' screenshot makes much sense, or is needed. There's no difference in citation style depending on which reference type it is.

**Comment by JGilbert-eLife** [comment]
"The way journal references are displayed depends on the referencing style." I'm not sure there's a lot gained by saying this. This is a page covering eLife's style, so it's probably better to just lay that out and avoid confusion by referencing other journals' styles.

**Comment by JGilbert-eLife** [comment]
"Yes (or a collaboration must be entered instead)" on Authors and Collaborations is potentially misleading. At least one author or collaboration is required, might be a better way of saying it.

**Comment by JGilbert-eLife** [comment]
`<article-title>`"XXXXXX" where XXXXXX is the article title - not sure why this has the XXXXXXs and the other elements don't?

**Comment by JGilbert-eLife** [comment]
Is it worth stating at the start that either e-location OR page range is allowed, but not both?

**Comment by JGilbert-eLife** [comment]
"If this is no expanded to the full" -> "If this is not expanded to the full"

**Comment by JGilbert-eLife** [comment]
"Many journals do not use volume numbers, page ranges or e-locations." Probably not relevant but the main distinction is that only print journals have page ranges.

**Comment by JGilbert-eLife** [comment]
"Searching for this reference online takes you to this page." I think it might be clearer to say "Searching for this reference online, one of the results is this page. "

**Comment by JGilbert-eLife** [comment]
Why is the Schematron section not split in the usual fashion between content and structure? If this split isn't applicable, I'd suggest not having a level 2 heading.

**Comment by JGilbert-eLife** [comment]
Also, I think it's 'Schematron warnings' as the level 1 heading on other pages - does that need fixing in the template?

**Comment by JGilbert-eLife** [comment]
Should there be a space in 'Inpress'? Does it show run-together on Kriya?

**Comment by JGilbert-eLife** [comment]
Regarding err-elem-cit-journal-6-2, if we have the final publication details, shouldn't we be including them even if that article is not yet live? The point of an in press label is to substitute for publication details that are not yet known. If we had the final details - page range, elocation, DOI etc - then it does more service to readers to include that than to remove them in favour of an in press comment that will become outdated within, say, a month.

**Comment by JGilbert-eLife** [comment]
Related to Melissa's comment above about journal-off-presence, it's probably worth an example to show which bit and how much of the extra text needs deleting.

**Comment by JGilbert-eLife** [comment]
handbook-presence - Does this account for reference with entries in PubMed (I think some of the 'handbooks' are treated like journals) and do we care if changing these to books loses us the PMID? It would also lose us any DOI, so probably worth clearly stating that the DOI will need to be re-added for it as a book.

**Comment by JGilbert-eLife** [comment]
article-title-correction-check - There are plenty of articles that just legitimately have 'correction' or even 'retraction' in their titles e.g. https://pubmed.ncbi.nlm.nih.gov/30037348/, https://doi.org/10.1055/s-0038-1672142. This rule fires for those too, so this needs to be covered.

**Comment by JGilbert-eLife** [comment]
Edge case - I have seen a couple of instances of preprint servers outside the common ones being captured as journals. It is sometimes possible to spot them if looking up references that don't have page ranges, volumes or e-locations. Is this worth covering?

**Comment by JGilbert-eLife** [comment]
I think that's everything from me! Thanks for tackling this, @naushinthomson!

**Comment by bcollins14** [comment]
I'm not sure if this has been covered but when validating the reference against crossref or Pubmed, should we mention that you do not need to select an article as it can be the case that it will not appear? I think the way it reads suggests you must pick an article from what the validator suggests.

**Comment by bcollins14** [comment]
**journal-off-presence
Warning: ref 'XXXXXX' has a source title which contains the text 'official journal' - 'XXXXXX'. Is this necessary?
Action: This warning will appear if a journal name includes the text 'official journal'. Delete this text from the journal name.** 

In the rare case that something like the 'Official Journal of the European Union' was mistagged, this would be wrong.. I just Googled this but I think this would apply!

**Comment by bcollins14** [comment]
Everything else I would have said, people have covered. Thanks! :)

**Comment by naushinthomson** [comment]
To cover in Production meeting:

- [ ] General ref rules should go in the 'References' gitbook page
- [ ] Do we search PubMed before Crossref in Kriya when adding refs?
- [ ] Run updated table by team
- [ ] Run updated 'Finding reference details' section by team. Do we need to add an example of a journal that doesn't use these details at all?
- [ ] Have added info about how to delete fields in Kriya - but this doesn't always work! Thoughts?
- [ ] Which fields need to be filled in to get the correct result and auto-populate of the rest of the fields?
- [ ] article-title-child-1 - Is the author query correct? If so it should be added to the other tests that ask you to look up the article online. Should you even look up the article online or can the formatting just be removed (what about correcting italicisation?)
- [ ] journal-preprint-check - Any suggestions for other preprints to add to this test
- [ ] online-journal-w-page - Any suggestions for online-only journals to add
- [ ] elife-ref-check - What should Exeter do if the paper hasn't been published yet?
- [ ] Run updated conference-check actions by team
- [ ] Can err-elem-cit-journal-2-1 fire if there are more than 1 author groups in the XML? (If so, action needs to be updated)
- [ ] err-elem-cit-journal-2-2 - How might this happen?
- [ ] err-elem-cit-journal-3-1 - Run updated action by team
- [ ] Adding the text 'if appropriate by looking up the article details online' to other actions - unsure what you mean, all actions that need it currently have a variation of 'look up the article online' I think?
- [ ] Changing 'look up the article online' to something like 'find the source for the journal title (either PubMed, PMC or the publisher website)' - thoughts?
- [ ] PNAS - Run updated action by team
- [ ] G3 - Why do we allow 'G3: Genes|Genomes|Genetics' AND 'G3: Genes, Genomes, Genetics'. Should it not just be one of these variants?
- [ ] Research-gate-check - run updated action by team
- [ ] journal-replacement-character-presence - is this just checking the journal name?
- [ ] article-title-correction-check - Adding context about why we do this?
- [ ] err-elem-cit-journal-6-2, 3 and 4 - What do we do for In Press articles in general? (Also check consistent capitalisation of 'in press'. 
- [ ] 'I have seen a couple of instances of preprint servers outside the common ones being captured as journals. It is sometimes possible to spot them if looking up references that don't have page ranges, volumes or e-locations. Is this worth covering?'
- [ ] handbook-presence - Does this account for reference with entries in PubMed (I think some of the 'handbooks' are treated like journals) and do we care if changing these to books loses us the PMID? It would also lose us any DOI, so probably worth clearly stating that the DOI will need to be re-added for it as a book.
- [ ] article-title-correction-check - will this only fire if 'correction, retraction, erratum' are at the beginning of the article title? If so, there are plenty of articles that just legitimately have 'correction' or even 'retraction' in their titles (and action needs to be updated). 
- [ ] journal-off-presence - run updated action by team
- [ ] Run split of content vs xml structure checks by team
